src/main/java/org/spongepowered/api/CatalogKey.java

```diff
@@ -82,6 +83,19 @@ static CatalogKey sponge(final String value) {
         return of(SPONGE_NAMESPACE, value);
     }
 
+    /**
+     * Creates a catalog key with the specified value and
+     * the namespace of the active plugin in the {@link CauseStackManager}.
+     *
+     * @param value The value
+     * @return A new catalog key
+     */
+    static CatalogKey forActivePlugin(final String value) {
```

Comment:
Is this name good?

Reply:
That signature doesn't reflect how this method works:

Key:
  namespace: id of plugin in cause stack
  value: value

I know that the name can be confusing, that's why I would like different options.

Reply:
Ah so
`namespaced(String)`
`of(String)`
`ofSelf(String)`
`forSelf(String)`